small fixes from my old PR 505

tkimnguyen opened the pull request:
https://github.com/plone/training/pull/505

stevepiercy commented on docs/plone_training_config/instructions_plone5.md:
````diff
@@ -58,12 +58,12 @@ sudo apt-get -y upgrade
 
 ```shell
-sudo apt-get install python3.7-dev python3.7-tk python3.7-venv build-essential libssl-dev libxml2-dev libxslt1-dev libbz2-dev libjpeg62-dev
+sudo apt-get install python3.8-dev python3.8-tk python3.8-venv build-essential libssl-dev libxml2-dev libxslt1-dev libbz2-dev libjpeg62-dev
````
Why not go up to python3.9?

tkimnguyen replied:
Yeah I thought about that.... 3.10 has just come out too. Should I go with that?

tkimnguyen replied:
all right, I'll just make it 3.9 and be done with it for today :)

stevepiercy replied:
I have a personal rule: "I shall not publish anything that I have not personally tried." IOW, don't make up 💩 .

If you have tried it and it works, then yes. Else, stick with what you know.

stevepiercy reviewed (approved):
I approve, but wonder why you went with python3.8 instead of python3.9?

stevepiercy commented on docs/plone_training_config/instructions.md:
````diff
@@ -83,7 +83,7 @@ If you feel comfortable, please work on your own machine with your own Python.
 ```
 
-The following instructions are based on Ubuntu and macOS, if you use a different operating system (OS), please adjust them to fit your OS.
+The following instructions are based on Ubuntu and macOS; if you use a different operating system (OS), please adjust them to fit your OS.
````
```suggestion
The following instructions are based on Ubuntu and macOS.
If you use a different operating system (OS), please adjust them to fit your OS.
```

stevepiercy commented on docs/plone_training_config/instructions.md:
````diff
@@ -136,7 +136,7 @@ Now you can run the buildout for the first time:
 ```
 
-This will take **very long** time and produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
+This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
````
```suggestion
This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages.
Once it is done, you can start your Plone instance with the following command.
```

stevepiercy commented on docs/plone_training_config/instructions_plone5.md:
````diff
@@ -46,7 +46,7 @@ you need [NodeJS](https://nodejs.org/en/download/) installed on your development
 ```
 
-The following instructions are based on Ubuntu and macOS, if you use a different operating system (OS), please adjust them to fit your OS.
+The following instructions are based on Ubuntu and macOS; if you use a different operating system (OS), please adjust them to fit your OS.
````
```suggestion
The following instructions are based on Ubuntu and macOS.
If you use a different operating system (OS), please adjust them to fit your OS.
```

stevepiercy commented on docs/plone_training_config/instructions_plone5.md:
````diff
@@ -78,19 +78,19 @@ mkdir training
 ```
 
-This will take **very long** time and produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
+This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
````
```suggestion
This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages.
Once it is done, you can start your Plone instance with the following command.
```

stevepiercy reviewed (changes requested):
- Minor grammar fixes.
- One sentence per line.
- Semicolons may be confusing to English as a second language readers. They should be avoided in favor of short sentences.

stevepiercy commented:
Links need fixing:
- https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2584 
- https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2110
- https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2122
- https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2361
- https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2363
- https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2749

stevepiercy commented:
@tkimnguyen although you did not break those links, would you please fix them? We have not yet stated who is responsible for fixing link rot: Author, Maintainer, Maintainer chasing down Author, Contributor of PR, or no one.

stevepiercy commented:
I also do not know what 👀  means in a reaction.

tkimnguyen commented:
Yup will do. Was out of commission this weekend. 👀 meant I was looking...

tkimnguyen commented:
> Links need fixing:
> 
>     * https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2584

This one works... their SSL cert has been updated

>     * https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2110

Fixed

>     * https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2122

The offending http://plone.org is no longer there.

>     * https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2361

Fixed

>     * https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2363

Fixed

>     * https://github.com/plone/training/runs/4499227483?check_suite_focus=true#step:5:2749

Fixed!

tkimnguyen commented:
OMG OMG y'all passed
